circle.yml

```diff
@@ -5,7 +5,7 @@ general:
 machine:
   # Add some environment variables
   environment:
-   HOST: $(case $CIRCLE_NODE_INDEX in 0) host='' ;; 1) host='i686-w64-mingw32-' ;; esac; echo $host)
+    CROSS_COMPILE: $([[ $CIRCLE_NODE_INDEX -eq 1 ]] && echo 'i686-w64-mingw32-')
 
```

Comment:
@pscollins sorry I should ask during this PR.

what was the motivation to change this line ?
because I'm going to revert this to add more host (arm) and it's hard to handle multiple conditions.

tentative patch is available here.

https://github.com/libos-nuse/lkl-linux/commit/be6d41f3b9b6fd1108fbabfab95d92c761b7a05d